### PR TITLE
TensorRT OSS 21.02 release

### PR DESCRIPTION
**Pull request by rajeevsrao**
Signed-off-by: Rajeev Rao <rajeevrao@nvidia.com>

### CODE REVIEW

**Review by ttyio — APPROVED** [review]

**Comment by ttyio** [comment]
@rajeevsrao  I see some python temp file *.pyc is added in the PR

**Review by ttyio — CHANGES_REQUESTED** [review]
Remove  demo/BERT/helpers/__pycache__/

File: demo/BERT/README.md [hunk 1]
```diff
@@ -89,58 +89,48 @@ The following section lists the requirements that you need to meet in order to r
...
+This demo BERT application can be run within the TensorRT OSS build container. If running in a different environment, following packages are required.
 
 * [NGC CLI](https://ngc.nvidia.com/setup/installers/cli) - for downloading BERT checkpoints from NGC.
 * PyPI Packages:
-  * [pycuda](https://pypi.org/project/pycuda/) 2019.1.2
-  * [onnx](https://pypi.org/project/onnx/1.6.0/) 1.6.0
-  * [tensorflow](https://pypi.org/project/tensorflow/1.15.3/) 1.15
-* NVIDIA [Volta](https://www.nvidia.com/en-us/data-center/volta-gpu-architecture/), [Turing](https://www.nvidia.com/en-us/geforce/turing/) or [Ampere](https://www.nvidia.com/en-us/data-center/nvidia-ampere-gpu-architecture/) based GPU with NVIDIA Driver 450.37 or later.
+  * [pycuda](https://pypi.org/project/pycuda/) (tested 2019.1.2)
+  * [onnx](https://pypi.org/project/onnx/1.6.0/) (tested 1.7.0)
```
Comment (kevinch-nv) [inline demo/BERT/README.md:97]: Link should be updated here

File: quickstart/IntroNotebooks/onnx_helper.py [hunk 2]
```diff
@@ -0,0 +1,72 @@
+import numpy as np
```
Comment (kevinch-nv) [inline quickstart/IntroNotebooks/onnx_helper.py:1]: missing cp header

File: quickstart/IntroNotebooks/helper.py [hunk 3]
```diff
@@ -0,0 +1,94 @@
+from tensorflow.python.compiler.tensorrt import trt_convert as tf_trt
```
Comment (kevinch-nv) [inline quickstart/IntroNotebooks/helper.py:1]: cp header

File: quickstart/IntroNotebooks/benchmark.py [hunk 4]
```diff
@@ -0,0 +1,34 @@
+import time
```
Comment (kevinch-nv) [inline quickstart/IntroNotebooks/benchmark.py:1]: cp header

File: quickstart/IntroNotebooks/Additional Examples/helper.py [hunk 5]
```diff
@@ -0,0 +1,94 @@
+from tensorflow.python.compiler.tensorrt import trt_convert as tf_trt
```
Comment (kevinch-nv) [inline quickstart/IntroNotebooks/Additional Examples/helper.py:1]: cp header

**Review by kevinch-nv — CHANGES_REQUESTED** [review]

**Review by kevinch-nv — APPROVED** [review]

**Review by ttyio — APPROVED** [review]

**Pull request merged**